src/components/button/__tests__/index.kit.spec.js

```diff
@@ -0,0 +1,142 @@
+import React, {useState} from 'react';
```

Comment:
This is the spec that checks the driver? maybe it's better to name the file `index.driver.spec.js`?

Reply:
I don't think it right to say that it checks the driver.
more like, it checks the button using the driver.
I still not sure what is good name for this, so I will change like you say